ci-operator/step-registry/ipi/install/install/ipi-install-install-commands.sh

```diff
@@ -4,4 +4,49 @@ set -o nounset
 set -o errexit
 set -o pipefail
 
-# TODO(skuznets): oh boy
+trap 'CHILDREN=$(jobs -p); if test -n "${CHILDREN}"; then kill ${CHILDREN} && wait; fi' TERM
+
+cluster_profile=/var/run/secrets/ci.openshift.io/cluster-profile
+export SSH_PRIV_KEY_PATH=${cluster_profile}/ssh-privatekey
+export PULL_SECRET_PATH=${cluster_profile}/pull-secret
+export OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE=${RELEASE_IMAGE_LATEST}
+export OPENSHIFT_INSTALL_INVOKER=openshift-internal-ci/${JOB_NAME_SAFE}/${BUILD_ID}
+export HOME=/tmp
+
+case "${CLUSTER_TYPE}" in
+aws)
+    export AWS_SHARED_CREDENTIALS_FILE=${cluster_profile}/.awscred
+    export AWS_REGION=us-east-1;;
+azure4) export AZURE_AUTH_LOCATION=${cluster_profile}/osServicePrincipal.json;;
+gcp)
+    export GCP_REGION=us-east1
+    export GCP_PROJECT=openshift-gce-devel-ci
+    export GOOGLE_CLOUD_KEYFILE_JSON=${cluster_profile}/gce.json;;
+esac
```

Comment:
Here's another `case` where [a `*)` guard][1] would be nice.

[1]: https://github.com/openshift/release/pull/6708#discussion_r370957816